stellargraph/utils/history.py

```diff
@@ -33,7 +33,12 @@ def plot_history(history, individual_figsize=(7, 4), **kwargs):
     Args:
         history: the training history, as returned by :meth:`tf.keras.Model.fit`
         individual_figsize (tuple of numbers): the size of the plot for each metric
+        return_figure (bool): if True, then the figure object with the plots is returned, None otherwise.
         kwargs: additional arguments to pass to :meth:`matplotlib.pyplot.subplots`
+
+    Returns:
+        [matplotlib.figure.Figure]: The figure object with the plots if `return_figure=True`, None otherwise
```

Comment:
Thanks for taking care to expand the docs, I think there's just a couple of minor tweak we can do here. Currently the [] brackets are appearing in the documentation which looks a bit peculiar, and could be misleading (someone might interpret it as meaning it returns a list `[...]` that contains a figure): https://stellargraph--1309.org.readthedocs.build/en/1309/api.html#stellargraph.utils.plot_history

<img width="565" alt="image" src="https://user-images.githubusercontent.com/1203825/80171447-c4b3f600-862d-11ea-8a94-562c0d2b60a0.png">

Additionally, unlike markdown, a single ` doesn't mean code in reStructured Text (notice how the `return_figure=True` text is italicised); it has to be doubled.

Thus, I think we should adjust this to:

```suggestion
        :class:`matplotlib.figure.Figure`: The figure object with the plots if ``return_figure=True``, None otherwise
```

Reply:
Thanks, of course we should change that. The thing with the brackets I got from here: [group_aggregate](https://stellargraph--1309.org.readthedocs.build/en/1309/api.html#stellargraph.layer.MeanAggregator.group_aggregate). I thought they are used if the type is not built-in.

Reply:
Ah, I think that's a bug there too. Thanks for noticing it! I opened https://github.com/stellargraph/stellargraph/issues/1364.